Readme.md

```diff
@@ -1,23 +1,98 @@
 <!-- default badges list -->
-![](https://img.shields.io/endpoint?url=https://codecentral.devexpress.com/api/v1/VersionRange/128535450/14.1.8%2B)
 [![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/T175881)
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
-<!-- default file list -->
-*Files to look at*:
 
-* [Default.aspx](./CS/Default.aspx) (VB: [Default.aspx](./VB/Default.aspx))
-* [Default.aspx.cs](./CS/Default.aspx.cs) (VB: [Default.aspx.vb](./VB/Default.aspx.vb))
-<!-- default file list end -->
-# ASPxGridView - How to set text wrapping in different ways
+# Grid Control for ASP.NET Web Forms - How to set text wrapping in different ways
```

Comment:
How to set text wrapping in different ways --> this phrasing won't work, can you rewrite this? Maybe Different options for wrapping text or something else...

Reply:
```suggestion
# Grid Control for ASP.NET Web Forms - Different options for wrapping text
```